src/events/events.service.ts

```diff
@@ -23,8 +23,8 @@ export class EventsService {
     return this.eventRepository.find({"eventId": eventId});
   }
 
-  findEventByHirerId(hirerId: number): Promise<Event[]> {
-    return this.eventRepository.find({userId: hirerId});
+  findEventByHirerId(userId: number): Promise<Event[]> {
+    return this.eventRepository.find({"userId": userId});
```

Comment:
The field of which key is the same as value shall be written as shorthand syntax.